<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1366</width>
      <height>768</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>100</y>
        <width>791</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_EventT">
       <item>
        <widget class="QFrame" name="EventT">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="EventT_TitleFrame">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="EventT_Title">
               <property name="text">
                <string>Weekly Schedule</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="EventT_Table">
            <row>
             <property name="text">
              <string>7:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7:00 PM</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Sunday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Monday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Tuesday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Wednesday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Thursday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Friday</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Saturday</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>278</width>
        <height>551</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="Layout_Events">
       <item>
        <widget class="QFrame" name="Events">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>Upcoming Events</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>Rescheduled Events</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft|Qt::AlignTop">
           <widget class="QLabel" name="DatePublished_Events">
            <property name="text">
             <string>Date Published</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QTableWidget" name="Events_table">
            <row>
             <property name="text">
              <string>7:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>10:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>11:00 AM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>12:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>1:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>6:00 PM</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7:00 PM</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>View Attendance</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>1271</width>
        <height>661</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QTableWidget" name="tableWidget">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>140</y>
            <width>1261</width>
            <height>461</height>
           </rect>
          </property>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Student ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Year</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Section</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Course</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender/Sex</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Attendance Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time IN</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time OUT</string>
           </property>
          </column>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>61</y>
            <width>291</width>
            <height>71</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Event: Event Name</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>Filter</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>1170</x>
            <y>600</y>
            <width>81</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>Go Back</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
